src/deploy/functions/runtimes/node/index.ts

```diff
@@ -80,10 +92,77 @@ export class Delegate {
     return Promise.resolve(() => Promise.resolve());
   }
 
+  serve(
+    port: number,
+    adminPort: number,
+    envs: backend.EnvironmentVariables
+  ): Promise<() => Promise<void>> {
+    const childProcess = spawn("./node_modules/.bin/firebase-functions", [this.sourceDir], {
+      env: {
+        ...envs,
+        PORT: port.toString(),
+        STACK_CONTROL_API_PORT: adminPort.toString(),
```

Comment:
Just STACK_CONTROL_API (or FUNCTIONS_CONTROL_API according to internal discussion) = true. Early prototypes spun up two ports, but Python cannot do this, so we're using __ instead.